### DISCUSSION TITLE
Docker pull failed with exit code 1 for: Pull ghcr.io/actions/jekyll-build-pages:v1.0.4

### DISCUSSION THREAD

**Discussion by kortina** [community/community#45835] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I am trying to deploy my website with GitHub pages and seeing the following error for the first time (after years of using this product):

<img width="903" alt="image" src="https://user-images.githubusercontent.com/5924/215604696-3dbe2797-ce29-472c-a4b4-0f666d45ebd8.png">

I tried "Re-run jobs" but continuously get the same error.

**Comment by yoannchaudet** [comment]
Related to https://www.githubstatus.com/incidents/zf2tk6x40gmh

**Comment by kortina** [comment]
Resolved.
